docs/conf.py

```diff
@@ -12,6 +12,16 @@
 
 import os
 import sys
+
+
+# Get version number
+version_path = os.path.join('../', 'kaolin', 'version.py')
```

Comment:
why not just `version_path = '../kaolin/version.py' ?

Reply:
Fixed. You are absolutely correct.